lib/features/giving_flow/screens/choose_amount_slider_screen.dart

```diff
@@ -0,0 +1,256 @@
+
+                              context
+                                  .read<CreateTransactionCubit>()
+                                  .changeAmount(value);
+
+                              // AnalyticsHelper.logEvent(
+                              //   eventName: AmplitudeEvent.amountPressed,
+                              //   eventProperties: {'amount': _selectedAmount},
+                              // );
+                            },
+                          ),
+                          Padding(
+                            padding: EdgeInsets.symmetric(horizontal: 15),
+                            child: Row(
+                              children: [
+                                Text(
+                                  "\$0",
+                                  style: TextStyle(
+                                    fontSize: 18,
+                                    color: Color(0xFF54A1EE),
+                                    fontWeight: FontWeight.bold,
+                                  ),
+                                ),
+                                Spacer(),
+                                Text(
+                                  "\$${state.maxAmaount.round()}",
+                                  style: TextStyle(
+                                    fontSize: 18,
+                                    color: Color(0xFF54A1EE),
+                                    fontWeight: FontWeight.bold,
+                                  ),
+                                ),
+                              ],
+                            ),
+                          ),
+                        ],
+                      ),
+                    ),
+                    SizedBox(
+                      height: 120,
+                    ),
+                  ],
+                ),
+              ),
+              floatingActionButtonLocation:
+                  FloatingActionButtonLocation.centerFloat,
+              floatingActionButton: Container(
+                height: 55,
+                margin: const EdgeInsets.only(bottom: 25),
+                child: Container(
+                  width: double.infinity,
+                  alignment: Alignment.center,
+                  padding: EdgeInsets.only(
+                    left: 40,
+                    right: 40,
+                  ),
+                  child: ElevatedButton(
+                    onPressed: state.amount == 0
+                        ? null
+                        : () async {
+                            if (state is CreateTransactionUploadingState) {
+                              return;
+                            }
+                            var transaction = Transaction(
+                              userId: _profilesCubit.state.activeProfile.id,
+                              collectGroupId:
+                                  '3fa85f64-5717-4562-b3fc-2c963f66afa6', // TODO use real value
```

Comment:
Does this need to be changed before merging?

Reply:
Yep, I updated it